src/submission/parseErrors/index.jsx

```diff
@@ -126,6 +127,30 @@ class ParseErrors extends Component {
   }
 }
 
+function errorResponseParser(errorResponse, uliNeeded) {
+  let apiErrorMessages =[]
+
+  errorResponse.forEach(function(errorsFound) {
+    if(errorsFound && errorsFound.errorMessages)
+    for (const [index, errorMessage] of errorsFound.errorMessages.entries()) {
+      let inputContent = errorMessage.inputValue
+
+      if (!errorMessage.inputValue) inputContent = <em>(blank)</em>
```

Comment:
This should likely be the following to guard against unexpected coercions:
```suggestion
      if (!errorMessage.inputValue === '') inputContent = <em>(blank)</em>
```